package.json

```diff
@@ -2,56 +2,54 @@
   "devDependencies": {
     "@esm-bundle/chai": "4.3.4",
     "@playwright/test": "^1.25.2",
-    "@testing-library/react": "12.1.2",
     "@types/chai": "^4.2.22",
     "@types/jest": "^27.0.2",
     "@types/node": "^16.11.4",
+    "@types/react": "^18.0.20",
+    "@types/react-router-dom": "^5.0.0",
     "@typescript-eslint/eslint-plugin": "^5.9.0",
     "@typescript-eslint/parser": "^5.9.0",
     "@vitejs/plugin-react": "^1.3.0",
-    "@web/test-runner": "^0.13.20",
-    "@web/test-runner-playwright": "^0.8.8",
-    "@web/test-runner-puppeteer": "^0.10.2",
+    "@web/test-runner": "^0.14.0",
+    "@web/test-runner-playwright": "^0.8.10",
     "chai": "^4.3.4",
-    "eslint": "8.6.0",
+    "eslint": "8.23.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "husky": "^7.0.4",
     "jest": "^29.0.2",
-    "playwright": "^1.16.1",
+    "playwright": "^1.25.2",
     "prettier": "2.4.1",
     "prettier-plugin-jsdoc": "^0.3.24",
-    "ts-jest": "^28.0.8",
-    "typedoc": "^0.22.15",
+    "react": "^18.2.0",
+    "ts-jest": "^29.0.1",
+    "typedoc": "^0.23.14",
     "typescript": "^4.8",
     "vite": "^3.0.5"
   },
   "name": "@tomic/root",
   "private": true,
   "scripts": {
-    "lint": "yarn workspaces foreach run lint",
-    "lint-fix": "yarn workspaces foreach run lint-fix",
-    "build": "yarn workspace @tomic/lib build && yarn workspace @tomic/react build && yarn workspace @tomic/data-browser build",
-    "build-server": "yarn run build && cp -r data-browser/dist/assets/ ../atomic-data-rust/server/app_assets/dist/ && cp data-browser/tests/e2e.spec.ts ../atomic-data-rust/server/e2e_tests/e2e-generated.spec.ts",
-    "test": "yarn workspaces foreach run test",
-    "test-query": "yarn workspace @tomic/data-browser run test-query",
-    "publish": "yarn workspaces foreach --no-private npm publish",
-    "start": "yarn workspace @tomic/lib watch & yarn workspace @tomic/react watch & yarn workspace @tomic/data-browser start",
-    "start-browser": "yarn workspace @tomic/data-browser start",
-    "start-lib": "yarn workspace @tomic/lib watch",
+    "lint": "pnpm run -r lint",
+    "lint-fix": "pnpm run -r lint-fix",
+    "build": "pnpm run -r build",
+    "build-server": "pnpm run build && cp -r data-browser/dist/assets/ ../atomic-data-rust/server/app_assets/dist/ && cp data-browser/tests/e2e.spec.ts ../atomic-data-rust/server/e2e_tests/e2e-generated.spec.ts",
+    "test": "pnpm run -r test",
+    "test-query": "pnpm run --filter @tomic/data-browser test-query",
+    "start": "pnpm run -r --parallel start",
     "typedoc": "typedoc . --options typedoc.json"
   },
   "workspaces": {
     "packages": [
       "lib",
       "react",
       "data-browser"
-    ]
+    ],
+    "packageManager": "pnpm@7.11.0"
```

Comment:
I think this should be at the root. Corepack does not recogonise it here but does when located at the root.